coralproject/cay
# Editing title and description

## What does this PR do?
- Fixes #474 

## How do I test this PR?
- Go to *Create form*
- Change the title and description and save --> You'll be redirected to *edit form*
- You should see your new title.
- Change the title and description again and save
- You should see your new title.
- Go to *Create form* again
- Change the title for the new form
- Click save: it should go to Edit with your new title
